packages/eslint-config-airbnb/index.js

```diff
@@ -1,13 +1,7 @@
-const reactRules = require('./react');
-const base = require('./base');
-
-// clone this so we aren't mutating a module
-const eslintrc = JSON.parse(JSON.stringify(base));
-
-// manually merge in React rules
-eslintrc.plugins = reactRules.plugins;
-Object.keys(reactRules.rules).forEach(function assignRule(ruleId) {
-  eslintrc.rules[ruleId] = reactRules.rules[ruleId];
-});
-
-module.exports = eslintrc;
+module.exports = {
+  'extends': [
+    // Need to qualify these here for ESLint to resolve them properly.
```

Comment:
`npm run lint` didn't work when referencing them relatively. It interpreted them as the names of `eslint-config-*` packages for some reason.

It's possible that doing something like https://github.com/walmartlabs/eslint-config-defaults/blob/master/package.json#L34 involving making `main` not be `index.js` could work, but I didn't try it, because this approach seemed tolerable.